cloudkeeper-executors/cloudkeeper-forking-executor/pom.xml

```diff
@@ -25,6 +25,28 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>analyze-only</goal>
+                        </goals>
+                        <configuration>
+                            <failOnWarning>true</failOnWarning>
+                            <outputXML>true</outputXML>
+                            <!-- Option ignoreNonCompile below is set true because maven-dependency-plugin complains
+                                 about unused dependency on slf4j-simple which is need to properly run the tests but not
+                                 needed to compile them and is not a transitive dependency of artifacts on which
+                                 execution of tests depends on. Setting this property to true resolves this problem, but
+                                 it is very blunt mechanism and you should be careful in this package declaring
+                                 non-compile dependencies lest you declare dependency that is not actually needed. -->
+                            <ignoreNonCompile>true</ignoreNonCompile>
```

Comment:
Yes, that is much better solution.

On Fri, Feb 19, 2016 at 11:51 AM, Florian Schoppmann <
notifications@github.com> wrote:

> In cloudkeeper-executors/cloudkeeper-forking-executor/pom.xml
> <https://github.com/cloudkeeper-project/cloudkeeper/pull/27#discussion_r53509627>
> :
>
> > +                <artifactId>maven-dependency-plugin</artifactId>
> > +                <executions>
> > +                    <execution>
> > +                        <goals>
> > +                            <goal>analyze-only</goal>
> > +                        </goals>
> > +                        <configuration>
> > +                            <failOnWarning>true</failOnWarning>
> > +                            <outputXML>true</outputXML>
> > +                            <!-- Option ignoreNonCompile below is set true because maven-dependency-plugin complains
> > +                                 about unused dependency on slf4j-simple which is need to properly run the tests but not
> > +                                 needed to compile them and is not a transitive dependency of artifacts on which
> > +                                 execution of tests depends on. Setting this property to true resolves this problem, but
> > +                                 it is very blunt mechanism and you should be careful in this package declaring
> > +                                 non-compile dependencies lest you declare dependency that is not actually needed. -->
> > +                            <ignoreNonCompile>true</ignoreNonCompile>
>
> This should be much simpler. I suggest to solve the issue in the same way
> as other CK subprojects do it. E.g.,
>
>                 <configuration>
>                     <usedDependencies>
>                         <usedDependency>org.slf4j:slf4j-simple</usedDependency>
>                     </usedDependencies>
>                 </configuration>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudkeeper-project/cloudkeeper/pull/27/files#r53509627>
> .
>